addons/web_dashboard/static/src/xml/dashboard.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<templates xml:space="preserve">
+    <t t-name="DashboardMain">
+        <div class="o_web_dashboard">
```

Comment:
A row must always have one of its parent which is a container. It's not the case here, so there is something wrong.